### PR TITLE
make icon for block configurable

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Same thing here, you have a commit not related to this PR.

**Comment by OskarStark** [comment]
updated @Soullivaneuh

**Comment by Soullivaneuh** [comment]
You still have 2 commits on this PR.

Squash them with fixup method, this should fix it.

**Comment by OskarStark** [comment]
fixxed @Soullivaneuh

**Comment by Soullivaneuh** [comment]
Not sure if this feature should be accepted or not, I let @rande review it.

BTW, as a new configuration key, documentation should be updated IMO.

**Comment by OskarStark** [comment]
where can i update the configuration?

**Comment by Soullivaneuh** [comment]
@OskarStark You mean the documentation?

Here: https://github.com/sonata-project/SonataTimelineBundle/tree/master/Resources/doc

**Comment by OskarStark** [comment]
done @Soullivaneuh

**Comment by Soullivaneuh** [comment]
OK thanks, let @rande decide for this one.

**Comment by rande** [comment]
Can you rebase this PR ? thanks

**Comment by OskarStark** [comment]
@rande @Soullivaneuh can we please merge this PR?

**Comment by Soullivaneuh** [comment]
@rande Should we not use same configuration data as other bundle? I mean full html string instead of icon class.

**Comment by OskarStark** [comment]
@rande please give feedback, so i can finish this PR

**Comment by OskarStark** [comment]
@Soullivaneuh i changed it to use the whole markup!

File: Block/TimelineBlock.php [hunk 1]
```diff
@@ -123,6 +126,7 @@ public function configureSettings(OptionsResolver $resolver)
         $resolver->setDefaults(array(
             'max_per_page'    => 10,
             'title'           => 'Latest Actions',
+            'icon'            => '<i class="fa fa-flag-o fa-fw"></i>',
```
Comment (Soullivaneuh) [inline Block/TimelineBlock.php]: Should be `fa fa-clock-o fa-fw`.

Comment (OskarStark) [inline Block/TimelineBlock.php]: oO strange, you are right!

Comment (OskarStark) [inline Block/TimelineBlock.php]: fixed!

**Comment by Soullivaneuh** [comment]
@OskarStark Could you add some tests?

**Comment by OskarStark** [comment]
sure, maybe tonight

**Comment by OskarStark** [comment]
rebased and added documentation

**Comment by OskarStark** [comment]
@Soullivaneuh i'm not really sure how/what to test this?

do have some idea?

Oskar

**Comment by Soullivaneuh** [comment]
Maybe you can get some inspiration here: https://github.com/sonata-project/SonataBlockBundle/tree/master/Tests/Block/Service

**Pull request merged**